### PR TITLE
updated readme with new notes

### PR DESCRIPTION
**Pull request by daniellefritz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,39 +3,55 @@
...
+- a button for authentication with github
+- a simple management system to keep track of event registrations
+- [need more details for this]
+
+Questions [have we answered any of these?]:
 - Where to host?
 - Who will do merge requests?
    - miguelcastillo
-   - daniellecloss
+   - daniellecloss [I am currently admin on the repo, due to starting the readme/notes, but it might be better to have someone more experienced handle the merge requests]
```
Comment (MiguelCastillo) [inline README.md]: I dont really think you need to be that explicit about letting someone handle pull requests...  It is sufficient to just called out yourself as a "contributor" ;)

**Comment by MiguelCastillo** [comment]
This is much better!!  I am glad we hashed out some details and added clarity to the project.  I think this is a good step that will continue to grow.  We could consider using something like trello to manage tasks. :)

@BrianGenisio @matthewmorgan - mind taking a look?

File: README.md (hunk 1 above)
Comment (daniellefritz) [inline README.md]: Fixed :)

File: README.md [hunk 2]
```diff
@@ -3,39 +3,53 @@
...
-Objectives:
+Project Objectives:
 - Let people collaborate on a project that has all the basic components of a production web application
 - Build a working web application
-- Core application should be very simple, like a todo app or something along those lines
+- learn github authentication
+- learn how CI (continuation integration) works
+- learn how to make a PR (pull request)
+- run unit tests
+- practive a code review process
```
Comment (MiguelCastillo) [inline README.md]: this might be a typo. is it `practice`?

File: README.md [hunk 3]
```diff
@@ -3,39 +3,53 @@
...
-- Basic scaffolding for a web system.
-- Repo/project name is negotiable!
-
-Objectives:
+Project Objectives:
 - Let people collaborate on a project that has all the basic components of a production web application
 - Build a working web application
-- Core application should be very simple, like a todo app or something along those lines
+- learn github authentication
+- learn how CI (continuation integration) works
```
Comment (matthewmorgan) [inline README.md]: I think this should be 'continuous' integration.

File: README.md [hunk 4]
```diff
@@ -3,39 +3,53 @@
...
+- learn how to make a PR (pull request)
+- run unit tests
```
Comment (matthewmorgan) [inline README.md]: very minor: reword to 'learn to write and run unit tests'?

**Comment by matthewmorgan** [comment]
Hey guys, I apologize-- I did not see the notification on this.  I get so many from work it got lost in the fray.

I did make a couple of comments on this, minor changes.  Thanks for writing it up!

**Comment by daniellefritz** [comment]
Thank you both, I'm feeling a bit silly at my typos...  But, eh, that's why we have code reviews, right? ;)

Let me know if there is anything else!

**Pull request merged**